src/features/loupe/BoundingBox.jsx

```diff
@@ -117,6 +122,7 @@ const BoundingBox = ({ imgId, imgDims, object, objectIndex, focusIndex, setTempO
   const catSelectorRef = useRef(null);
   const focusRef = useRef(null);
   const dispatch = useDispatch();
+  const dragRef = useRef(null);
```

Comment:
All changes made to this file were to address some warnings showing on the console.

Reply:
related: https://github.com/react-grid-layout/react-draggable/pull/478

Reply:
related: https://github.com/react-grid-layout/react-draggable/pull/478

Reply:
@nathanielrindlaub not entirely sure what line 280 does, but still causes a warning

Reply:
removed these "fixes"